tests/callbacks/test_callbacks.py

```diff
@@ -214,18 +200,12 @@ def on_test_end(self, trainer, pl_module):
 
 def test_early_stopping_no_val_step(tmpdir):
     """Test that early stopping callback falls back to training metrics when no validation defined."""
-    class ModelWithoutValStep(LightTrainDataloader, TestModelBase):
-
-        def training_step(self, *args, **kwargs):
-            output = super().training_step(*args, **kwargs)
-            loss = output['loss']  # could be anything else
-            output.update({'my_train_metric': loss})
-            return output
-
-    model = ModelWithoutValStep(tutils.get_default_hparams())
 
-    stopping = EarlyStopping(monitor='my_train_metric', min_delta=0.1)
+    model = EvalModelTemplate(tutils.get_default_hparams())
+    model.validation_step = None
```

Comment:
do we want this to be `None` or do we want `model.validation_step = LightningModule.validation_step`?

Reply:
I had it also originally with `LightningModule.validation_step` but @williamFalcon commented that he prefers none... For me `LightningModule.validation_step` would be easier to understand that it is not implemented...